notification-server/README.md

```diff
@@ -21,4 +21,26 @@ and run the application with
 cargo run --bin <BINARY_NAME>
 ```
 
-where `<BINARY_NAME>` is the name of the binary you want to run.
\ No newline at end of file
+where `<BINARY_NAME>` is the name of the binary you want to run.
+
+## API subscribe documentation
+
+The subscription endpoint is idempotent.
+
+All account to device mapping being registered under a single endpoint call will have the same preferences
+Accounts is a list of base58 encoded account ids.
+
+At most 1000 preferences and accounts can be registered in one call and only a 1000 accounts be queried at once.
+
+Should conflicts occur upon subscription updates, then only the preferences becomes updates.
+
+### Example:
+
+```shell
+❯ curl -X PUT "http://localhost:3030/api/v1/device/example-device/subscription" \
+    -H "Content-Type: application/json" \
+    -d '{
+        "preferences": ["CIS2", "CCDTransaction"],
```

Comment:
CIS2 is an event by itself right? CCD is a coin. 

Imo we are then referring to a coin and an event, which seems like a mistake.

Reply:
Renaming to CIS2Transaction because it aligns with documentation: https://concordium.atlassian.net/wiki/spaces/EN/pages/1502019590/Message+data+payload